river/tree/_nodes/base.py

```diff
@@ -145,7 +145,7 @@ def describe_subtree(self, tree, buffer: List[str], indent: int = 0):
                     text += f"{target_id}: {self.stats[target_id].mean} | {self.stats[target_id]}"
                     text += ", " if i < len(self.stats) - 1 else ""
             else:  # Single-target regression
-                text += f"{self.stats.mean} | {self.stats}"
+                text += f"{self.stats.mean.get()} | {self.stats.get()}"
```

Comment:
Could you instead do `{repr(self.stats.mean)} | {repr(self.stats)`. This was we will also get the `Mean` and `Var` labels.

Reply:
Good catch!

Reply:
I tried {repr(self.stats.mean)} | {repr(self.stats)} ` and got this:
```
Leaf = Output {Mean () | Var (
  ddof=1
)}
```

{repr(self.stats.mean.get())} | {repr(self.stats.get())} `  gives values only. 

I've hardcoded the labels like so `f"Mean={self.stats.mean.get()} | Var={self.stats.get()}"` 
to have output like this:
`Leaf = Output {Mean=0.0 | Var=0.0}`

as a short-term solution. 

Thoughts?

Reply:
I can confirm that the repr call is not working as we would expect.

Reply:
Here's our culprit:

https://github.com/online-ml/river/blob/de68efe1afed3f95344674e613abad5146a9541c/river/stats/base.py#L20

Univariate should inherit from `Statistic`. Do you confirm this, @MaxHalford?

Reply:
Good catch!

Reply:
I'll submit a fix. Once it is in place, I'll ask you, @ColdTeapot273K to pull the changes.

Reply:
@ColdTeapot273K, please pull the latest changes from `master`, then you can use Max's suggestion and everything should work fine.

Thanks once again for checking that!